freezeyt/freezer.py

```diff
@@ -525,6 +550,14 @@ async def handle_content_task(self, task, content):
         self.done_tasks[task.path] = task
         self.call_hook('page_frozen', hooks.TaskInfo(task))
 
+    @needs_semaphore
+    async def handle_file_task(self, task, path):
```

Comment:
Dont we consolidate this func and `handle_content_task`, to just handle task? Or it is another PR?